concepticondata/conceptlists/Marrison-1967-909.tsv

```diff
@@ -1,909 +1,909 @@
-Marrison-1967-883	883	witness	1144	WITNESS	2
-Marrison-1967-884	884	wolf	522	WOLF	2
-Marrison-1967-885	885	woman	962	WOMAN	2
-Marrison-1967-886	886	wonderful			
-Marrison-1967-887	887	wood (timber)	1803	WOOD	4
-Marrison-1967-888	888	word	1599	WORD	2
-Marrison-1967-889	889	work	984	WORK (LABOUR)	2
-Marrison-1967-890	890	world	965	WORLD	2
-Marrison-1967-891	891	worm	1219	WORM	2
-Marrison-1967-892	892	worship	257	WORSHIP	2
-Marrison-1967-893	893	wound	1109	WOUND	2
-Marrison-1967-894	894	wrap	1113	WRAP	2
-Marrison-1967-895	895	wrist	799	WRIST	2
-Marrison-1967-896	896	write	1672	WRITE	2
-Marrison-1967-897	897	wrong	1390	WRONG	2
-Marrison-1967-898	898	yam	410	YAM	2
-Marrison-1967-899	899	year	1226	YEAR	2
-Marrison-1967-900	900	yeast	967	YEAST	4
-Marrison-1967-901	901	yellow	1424	YELLOW	2
-Marrison-1967-902	902	yes	1268	YES	2
-Marrison-1967-903	903	yesterday	1174	YESTERDAY	2
-Marrison-1967-904	904	yoke	14	YOKE	2
-Marrison-1967-905	905	you	1213	YOU	2
-Marrison-1967-906	906	you two	2111	YOU TWO	2
-Marrison-1967-907	907	young	1207	YOUNG	2
-Marrison-1967-908	908	young man	76	YOUNG MAN	2
-Marrison-1967-909	909	young woman	77	YOUNG WOMAN	2
+Marrison-1967-909-1	1	among			
+Marrison-1967-909-2	2	and	1577	AND	2
+Marrison-1967-909-3	3	angry	1826	ANGRY	2
+Marrison-1967-909-4	4	animal	619	ANIMAL	2
+Marrison-1967-909-5	5	ankle	803	ANKLE	2
+Marrison-1967-909-6	6	answer	1275	ANSWER	2
+Marrison-1967-909-7	7	ant (black)			4
+Marrison-1967-909-8	8	ant (generic)	587	ANT	4
+Marrison-1967-909-9	9	ant (red)			4
+Marrison-1967-909-10	10	ant (white), termite	883	TERMITE	4
+Marrison-1967-909-11	11	arm	1673	ARM	2
+Marrison-1967-909-12	12	armlet			
+Marrison-1967-909-13	13	arrive	1387	ARRIVE	2
+Marrison-1967-909-14	14	arrow	977	ARROW	2
+Marrison-1967-909-15	15	ascend	705	GO UP (ASCEND)	2
+Marrison-1967-909-16	16	ashes	646	ASH	2
+Marrison-1967-909-17	17	ask (a question)	1309	ASK (INQUIRE)	4
+Marrison-1967-909-18	18	ask for	1828	ASK (REQUEST)	2
+Marrison-1967-909-19	19	at	1461	AT	2
+Marrison-1967-909-20	20	aunt	1272	AUNT	2
+Marrison-1967-909-21	21	awake / awaken	1602	WAKE UP	4
+Marrison-1967-909-22	22	axe	677	AXE	2
+Marrison-1967-909-23	23	baby	1289	BABY	2
+Marrison-1967-909-24	24	back (body)	1291	BACK	4
+Marrison-1967-909-25	25	bad	1292	BAD	2
+Marrison-1967-909-26	26	bag	2468	BAG	2
+Marrison-1967-909-27	27	bamboo	1927	BAMBOO	2
+Marrison-1967-909-28	28	banana	868	BANANA	2
+Marrison-1967-909-29	29	bank (edge)	113	SHORE	4
+Marrison-1967-909-30	30	bark (of tree)	1204	BARK	2
+Marrison-1967-909-31	31	basket (back, man's)			4
+Marrison-1967-909-32	32	basket (back, woman's)			4
+Marrison-1967-909-33	33	basket (generic)	1539	BASKET	4
+Marrison-1967-909-34	34	basket (large, storage)			4
+Marrison-1967-909-35	35	bat (animal)	1793	BAT	4
+Marrison-1967-909-36	36	be	1579	BE	2
+Marrison-1967-909-37	37	bead	1860	BEAD	2
+Marrison-1967-909-38	38	beam	1132	BEAM	2
+Marrison-1967-909-39	39	bean	832	BEAN	2
+Marrison-1967-909-40	40	bear (animal)	1820	BEAR	4
+Marrison-1967-909-41	41	bear (fruit)			4
```

Comment:
I suppose: leave both unlinked, (41 and the next). Too shaky what they mean with the meaning. And if you link one, you also have to link the other one, which makes it complicated.

Reply:
ok, and by the way, I figure out that cut with axe actually is CHOP, so I will change it as well.